components/PageView/PageView.tsx

```diff
@@ -4,21 +4,16 @@ import Arrow from '../Icons/DownArrow';
 import throttle from 'lodash/throttle';
 
 interface Props {
-  created: string;
-  title: string;
-  type: string;
-  date: string;
-  team: string;
-  note: string;
+  formData: {
+    created: string;
+    title: string;
+    type: string;
+    date: string;
+    team: string;
+    note: string;
+  };
 }
-const PageView = ({
-  created,
-  title,
-  type,
-  date,
-  note,
-  team,
-}: Props): React.ReactElement => {
+const PageView = ({ formData }: Props): React.ReactElement => {
```

Comment:
we should create a type `HistoryCaseFromData` as `https://github.com/LBHackney-IT/lbh-social-care/blob/main/types.ts#L44` and used it here.